client/src/components/Form/Elements/FormDrilldown/FormDrilldown.vue

```diff
@@ -96,3 +108,30 @@ function onSelectAll(selected: boolean): void {
             :handle-click="handleClick" />
     </div>
 </template>
+
+<style lang="scss">
+@import "theme/blue.scss";
+@import "~@fortawesome/fontawesome-free/scss/_variables";
+.ui-form-field {
+    div > .ui-drilldown {
+        $ui-drilldown-padding: 1rem;
+        $ui-drilldown-border: 0.5px solid $gray-500;
+
+        .indent {
+            padding-left: calc($ui-drilldown-padding + $ui-drilldown-padding/2);
+        }
+        .align-indent {
+            display: inline-block;
+            width: $ui-drilldown-padding;
+            border-bottom: $ui-drilldown-border;
+        }
+        .align-checkbox {
+            width: $ui-drilldown-padding;
+        }
+        /* selector: all except first nested drilldown */
+        & > * .descendant-lines {
+            border-left: $ui-drilldown-border;
+        }
+    }
+}
+</style>
```

Comment:
Please move these styles to the component which uses the `ui-drilldown` class. That way you can also use scoping, and the selectors can be much simpler.

Reply:
@ElectronicBlueberry Once again, I appreciate you explaining the "purpose" behind the change which allows me to address the issue on a deeper level. Let me know if [this commit](https://github.com/galaxyproject/galaxy/pull/18074/commits/1edab2f124cfc0588fe998e9fcd93c31a1bacaab) is more along the lines of what you had in mind.